pkgs/development/mobile/androidenv/compose-android-packages.nix

```
@@ -4,7 +4,7 @@
 
 { toolsVersion ? "26.1.1"
 , platformToolsVersion ? "31.0.3"
-, buildToolsVersions ? [ "31.0.0" ]
+, buildToolsVersions ? [ "32.0.0" ]
 , includeEmulator ? false
 , emulatorVersion ? "30.6.3"
```

Would be good to get this merged for API level 32 support!

```suggestion
, emulatorVersion ? "31.2.2"
```